pkg/cache/clusterqueue.go

```diff
@@ -434,7 +449,14 @@ func (c *clusterQueue) reportActiveWorkloads() {
 func (c *clusterQueue) updateWorkloadUsage(wi *workload.Info, m int64) {
```

Comment:
we should try to get rid of this function if the `m` value is used to fork, as opposed to be used as a multiplication factor.

Reply:
I agree, but it is outside of the scope of this PR. Let's fix in a followup to keep this PR's size reasonable.